components/identity-core/org.wso2.carbon.identity.core/pom.xml

```
@@ -82,6 +82,24 @@
             <groupId>org.wso2.carbon</groupId>
             <artifactId>org.wso2.carbon.user.core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.organization.management</groupId>
```

```suggestion
            <groupId>org.wso2.carbon.identity.organization.management.core</groupId>
```
Follow this PR https://github.com/wso2/carbon-identity-framework/pull/4215/files and remove exclusions tags